metricbeat/module/consul/_meta/fields.yml

```diff
@@ -0,0 +1,11 @@
+- key: consul
+  title: "consul"
+  description: >
+    beta[]
```

Comment:
You don't have to add it in the description but set `release: beta` flag here in the module and in the metricset fields.yml

Reply:
I think I finally understood, thank you for your patience @ruflin